File: day_16.js
const prompt = require("prompt-sync")({sigint: true})

function get_integer_array(input){
    new_array = []
    for (let count = input; count > 0; count--){
        new_array.push(count);
    }
    return new_array
}

let user_input = prompt("Enter a number: ");
console.log(get_integer_array(user_input));

File: Day16.java
import java.util.Arrays;

public class Day16 {
    public static int[] integerOfArray(int input){
        int[] newArray = new int[input];
        for (int count = input; count > 0; count--){
            newArray[5-count] = count;
        }
        return newArray;
    }
    public static void main(String[] args){
        int userInput =  5;
        System.out.print(Arrays.toString(integerOfArray(userInput)));

    }
}
